test/k8sT/Nightly.go

```diff
@@ -122,57 +151,29 @@ var _ = Describe("NightlyEpsMeasurement", func() {
 			}, endpointsTimeout, 3*time.Second).Should(BeTrue())
 		})
 		log.WithFields(logrus.Fields{"endpoint creation time": runtime}).Info("")
-	}, 1)
-
-	It("Should be able to connect from client pods to services while cilium pod is being restarted", func() {
-		defer kubectl.Delete(vagrantManifestPath)
-
-		connectivityTestsFinished := make(chan struct{})
-
-		// Run connectivity tests
-		go func() {
-			concurrency := 5
-			sem := make(chan struct{}, concurrency)
 
-			for serverIndex := 0; serverIndex <= lastServer; serverIndex++ {
-				for clientIndex := lastServer + 1; clientIndex < endpointCount; clientIndex++ {
-					sem <- struct{}{}
-					go func(to, from int) {
-						defer func() { <-sem }()
-						defer GinkgoRecover()
-
-						result := kubectl.TestConnectivityPodService(fmt.Sprintf("app%d", from), fmt.Sprintf("app%d-service", to))
-						result.ExpectSuccess(result.GetDebugMessage())
-
-					}(serverIndex, clientIndex)
+		services := getServices()
+		Expect(len(services)).To(BeNumerically(">", 0), "Services are empty and it's should not")
```

Comment:
Change to - "Was not able to get services" - saying it 'should not' in an Expect is redundant